n0rbs-cebu-singapore.md

```diff
@@ -0,0 +1,5 @@
+## Mandaue City, Cebu :smile:
+
+#### Waterfalls
```

Comment:
@wintercl133 one of the smaller falls. Southern part of the Philippines :)
  <img width="599" alt="screen shot 2016-06-22 at 12 58 10 am" src="https://cloud.githubusercontent.com/assets/2144028/16238795/7594acec-3814-11e6-9598-686863652dc2.png">